web/olga/analytics/tests/test_models.py

```diff
@@ -143,7 +143,13 @@ def test_overall_counts(self, mock_get_last_calendar_day):
         Verify that overall_counts method returns overall statistics instance counts in dict for previous calendar day.
         """
         mock_get_last_calendar_day.return_value = date(2017, 6, 1), date(2017, 6, 2)
-        test_result = dict(instances_count=0, courses_count=0, students_count=0, generated_certificates_count=0)
+        test_result = dict(
+            instances_count=0,
+            courses_count=0,
+            students_count=0,
+            generated_certificates_count=0,
+            registered_students_count=0
```

Comment:
I would add trailing comma to the last item in the dict to avoid future line updating is additional line will be added to the object